docs/release_notes/6.0.6-5.md

```diff
@@ -0,0 +1,33 @@
+# Percona Server for MongoDB 6.0.6-5 (2023-05-)
+
+| Release date | May , 2023|
+|------------- | ---------------|
+| **Installation** | [Installing Percona Server for MongoDB](../install/index.md) |
+
+
+Percona Server for MongoDB 6.0.6-5 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 6.0.6 Community Edition.
+
+It is rebased on [MongoDB 6.0.6 Community edition](https://www.mongodb.com/docs/manual/release-notes/6.0/#6.0.6---mar-13--2023) and supports the upstream protocols and drivers.
```

Comment:
I don't think "rebased" is correct word for PSMDB. Can we return to "based" wording?

Reply:
Done